data/assets/examples/debugcoordinateaxes.asset

```diff
@@ -1,6 +1,8 @@
 local transforms = asset.require('scene/solarsystem/planets/earth/transforms')
 local debugHelper = asset.require('util/debug_helper')
+local sun_transforms = asset.require('scene/solarsystem/sun/transforms')
 
+local sunRadius = 6.957E8
```

Comment:
These kinds of constants are duplicated in many files. Maybe we can create a helper asset that exports them? :)